CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog - Badger Components React
 
+## 00.7.0 (Nov 7, 2019)
```

Comment:
Update date + remove the extra leading `0`

Reply:
4a202ac

Reply:
186511f